### ISSUE TITLE
$PKG_SERVER URL gives 400

### ISSUE THREAD

**Issue by socialmedialabs** [holzschu/a-Shell-commands#48]
As the title says, when entering `pkg search` I get 
```
Packages available:
Not Found
```

When directly calling the URL https://raw.githubusercontent.com/holzschu/a-Shell-commands/master/ it yields an error 400, invalid request.

**Comment by holzschu** [comment]
When entering `pkg search`, I do get the list of available packages, so I don't understand why you got "Not found".

You cannot acces the directory  https://raw.githubusercontent.com/holzschu/a-Shell-commands/master/ directly (`curl https://raw.githubusercontent.com/holzschu/a-Shell-commands/master/` will give an error 400) but you can access specific files in there, for example `curl https://raw.githubusercontent.com/holzschu/a-Shell-commands/master/list` works on my laptop.

The short version is: I don't understand what is happening, but I suspect some issues on the network is more likely than a bug in `pkg`.
